core-plugins/src/main/java/co/cask/hydrator/plugin/batch/joiner/Joiner.java

```diff
@@ -52,6 +58,11 @@
   "records from non-required inputs will only be present if they match join criteria. If there are no required " +
   "inputs, outer join will be performed")
 public class Joiner extends BatchJoiner<StructuredRecord, StructuredRecord, StructuredRecord> {
+
+  public static final String JOIN_OPERATION_DESCRIPTIOPN = "Join Operation";
+  public static final String IDENTITY_OPERATION_DESCRIPTION = "Identity output as a part of JOIN";
```

Comment:
I find 'Identity output' confusing. 'Unchanged as part of a join'

Reply:
fixed.